tiledb/ml/models/model_list.py

```diff
@@ -0,0 +1,53 @@
+import tiledb.cloud
+
+FILETYPE_ML_NODEL = "ml_model"
+
+
+class ModelList:
```

Comment:
Do we have any unit tests for the following?

Reply:
No, we will need to call the rest server in the test for this or mock can we skip for a following PR?